inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -902,12 +902,30 @@ def normalize_collaborations(obj, eng):
             if not response:
                 LOGGER.info(u"(wf: %s) collaboration normalization: no match for: %s", obj.id, collaboration['value'])
                 continue
+            elif len(response.hits) > 1:
+                matched_collaboration_names = [
+                    matched_collaboration.collaboration.value for matched_collaboration in response
+                ]
+                LOGGER.info(
+                    u"(wf: {0}) ambiguous match for collaboration {1}. Matched collaborations: {2}".format(
+                        obj.id, collaboration['value'], ', '.join(matched_collaboration_names)
+                    )
+                )
+                continue
```

Comment:
this is the branch where you normalize based on a subgroup name, not based on the name of the full collaboration. The flow is a bit complex, but also there it should be unique.

Reply:
So it's the wrong place or logging should list subgroup names instead of collaborations?

Reply:
you should make it clear that it's about collaboration subgroups (`ambiguous match for collaboration subgroup`), and you should also add a check for several results to the non-subgroup matching.